test/built-ins/Iterator/prototype/Symbol.iterator/length.js

```diff
@@ -19,12 +19,9 @@ features: [Symbol.iterator]
 includes: [propertyHelper.js]
```

Comment:
All the tests in this directory need the `iterator-helpers` feature as well for `Iterator.prototype`.

Reply:
I guess I should revert back to deriving it.

```js
var IteratorPrototype = Object.getPrototypeOf(
  Object.getPrototypeOf([][Symbol.iterator]())
);
```

Then we don't need the flag.